packages/venia-ui/lib/components/MiniCart/ProductList/item.js

```diff
@@ -72,6 +72,7 @@ const Item = props => {
                             ? configured_variant.thumbnail.url
                             : product.thumbnail.url
                     }
+                    data-cy="item-image"
```

Comment:
We always start with upper case for `data-cy` 
```suggestion
                    data-cy="Item-image"
```